JustSaying/JustSayingFluentlyExtensions.cs

```diff
@@ -12,6 +14,8 @@ public static class JustSayingFluentlyExtensions
     {
         public static IMayWantOptionalSettings InRegion(this JustSayingFluentlyLogging logging, string region) => InRegions(logging, region);
 
+        public static IMayWantOptionalSettings InRegions(this JustSayingFluentlyLogging logging, IEnumerable<string> regions) => InRegions(logging, regions.ToArray());
```

Comment:
I suppose that would just create a stack overflow without a cast, but a cast would be more lightweight than a copy?